### DISCUSSION TITLE
If you turn off the GoPro while the program is running, it automatically turns on again.

### DISCUSSION THREAD

**Discussion by kizmonster** [gopro/labs#456] [General]
Hi.
I am developing a program using open gopro in C#, and if I turn off the gopro power while the program is running, the power turns on automatically again, but I don't know why.
If there is any way, please let me know.
A code sample would be even better.
The code that implements live streaming is still problematic. If possible, please give me the code as well. The API is not properly implemented.
thank you

**Comment by dnewman-gpsw** [comment]
This is Labs, not Open GoPro.  If you looking for Labs extensions that control WAKE on power, look here: https://gopro.github.io/labs/control/extensions/
